indi-qhy/qhy_ccd.cpp

```diff
@@ -1195,6 +1193,8 @@ bool QHYCCD::setupParams()
 {
 
 
+    LOG_DEBUG("setup params\n");
```

Comment:
This is not very useful log, please comment out. Also, remove any "/n" from other logs since it's not needed.

Reply:
Sheesh, agreed! Newlines are also removed. 
I wasn't too sure about the ones that are in IDLog(..) and left those in in the code. Let me know if those should go, too.